### PR TITLE
Add second Copilot video and more cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds second Copilot video per @banant, and more cleanup.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-228

### CODE REVIEW

File: docs/search/copilot.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: copilot
 title: Sumo Logic Copilot - Feature Preview
 sidebar_label: Copilot 🤖
+description: Streamline your log analysis with Sumo Logic Copilot, our AI-based assistant designed to simplify log analysis by allowing you to ask questions in plain English and providing search suggestions without the need to write log queries.
```
Comment (jpipkin1) [inline docs/search/copilot.md:5]:
```suggestion
description: Streamline your log analysis with Sumo Logic Copilot, our AI-based assistant. Copilot allows you to ask questions in plain English and provides search suggestions, all without the need to write log queries.
```

File: docs/search/copilot.md [hunk 2]
```diff
@@ -17,18 +18,38 @@ keywords:
 import Iframe from 'react-iframe';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-:::info
+<p><a href="/docs/beta"><span className="beta">Preview Release</span></a></p>
 This is a Preview release. To learn more, contact your Sumo Logic account executive.
-:::
 
-Sumo Logic Copilot is an AI-based assistant that streamlines log analysis insights by allowing you to ask questions in plain English and provides search suggestions, all without your having to write a log query.
+Sumo Logic Copilot is an AI-based assistant designed to simplify log analysis by allowing you to ask questions in plain English and providing search suggestions without the need to write log queries.
```
Comment (jpipkin1) [inline docs/search/copilot.md:24]:
```suggestion
Sumo Logic Copilot is an AI-based assistant designed to simplify log analysis. It allows you to ask questions in plain English and provides search suggestions, all without the need to write log queries.
```

File: docs/search/copilot.md [hunk 3]
```diff
@@ -53,29 +73,57 @@ In the scenario depicted in the video, we leverage Copilot to investigate a secu
 1. Add the security `groupid` to the prompt to tabulate events containing that data.
 1. Focus on a specific access key, `ABCDEFGOYCM3PIKNOVRA`, noting `PutRolePolicy` events indicating permission elevation. Multiple AWS accounts are impacted.
 1. We conclude that the access key was used to modify permissions, providing the attacker with a potential network entry point. Remediation would involve disabling the key, blocking the source IP, and further log analysis.
-</details>
 
-## Step 1: Open Copilot
+### Security example
```
Comment (jpipkin1) [inline docs/search/copilot.md:77]: I'm confused why you are including this video. It is the exact same example as the previous one, but presented without audio, and with visual highlighting of the steps.

Comment (kimsauce) [reply docs/search/copilot.md:77]: Same here! I've asked Bashyam about this in Slack.

Comment (kimsauce) [reply docs/search/copilot.md:80]: I've merged this branch (`copilot-vid`) into the `ai-machine-learning` branch. I'll work with Bashyam to sort out the duplicate video issue in [this PR](https://github.com/SumoLogic/sumologic-documentation/pull/3793).
